AK/StringBuilder.cpp

```diff
@@ -53,6 +53,13 @@ ErrorOr<void> StringBuilder::try_append(char ch)
     return {};
 }
 
+ErrorOr<void> StringBuilder::try_append(char ch, size_t n)
+{
+    for (size_t i = 0; i < n; ++i)
+        TRY(try_append(ch));
+    return {};
+}
+
```

Comment:
I'm not sure "`try_append()`" is the appropriate name for it. Maybe call it something like `try_append_repeated()`?